ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/models/Client.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.java.models;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3autogen.java.helpers.JavaHelper;
+import com.spectralogic.ds3autogen.utils.Helper;
+
+public class Client {
+
+    private final String packageName;
+    private final ImmutableList<Command> commands;
+    private final Helper helper;
+    private final JavaHelper javaHelper;
+
+    public Client(
+            final String packageName,
+            final ImmutableList<Command> commands) {
+        this.packageName = packageName;
+        this.commands = commands;
+
+        helper = Helper.getInstance();
+        javaHelper = JavaHelper.getInstance();
```

Comment:
After looking at it again, the helper and javaHelper are not used in the client templates. I'll just remove them.